src/raftstore/store/config.rs

```diff
@@ -135,6 +138,7 @@ impl Default for Config {
             messages_per_tick: 4096,
             max_peer_down_duration: ReadableDuration::minutes(5),
             max_leader_missing_duration: ReadableDuration::hours(2),
+            region_size_expired_time: ReadableDuration::hours(1),
```

Comment:
Is this too long?

Reply:
I think 10min should be OK?

Reply:
If there are 1TB data, it means TiKV will open all these files very 10~20 minutes. Is this ok?

Reply:
Oh, seems not ok, in that case I think 1 hour is reasonable :)

Reply:
so every 1h, we may open all files, this can hurt the performance?